covid19_rsid_export.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+import argparse
+import datetime
+import json
+import os
+import pymongo
+
+from ebi_eva_common_pyutils.mongo_utils import get_mongo_uri_for_eva_profile
+from pymongo import MongoClient
+from pymongo.read_concern import ReadConcern
+from pymongo.read_preferences import ReadPreference
+from typing import List
+
+
+timestamp = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+
+BATCH_SIZE = 1000
+ACCESSIONING_DATABASE_NAME = "eva_accession_sharded"
+RELEASE_RECORD_COLLECTION_NAME = "releaseRecordEntity"
+COVID19_ASSEMBLY_ACCESSION = "GCA_009858895.3"
+
+
+# See https://stackoverflow.com/a/57802672
+def _as_batch(cursor, batch_size=BATCH_SIZE):
+    # iterate over something (pymongo cursor, generator, ...) by batch.
+    # Note: the last batch may contain less than batch_size elements.
+    batch = []
+    try:
+        while True:
+            for _ in range(batch_size):
+                batch.append(next(cursor))
+            yield batch
+            batch = []
+    except StopIteration:
+        if len(batch):
+            yield batch
+
+
+def _get_allele_str(alleles: str) -> str:
+    return alleles if alleles else '-'
```

Comment:
Are REF or ALT are not present when we have deletion or insertion ? 
We should check with Ensembl/Covid19 if they are expecting the VCF convention.

Reply:
Added to Covid-19 DP [discussion notes](https://docs.google.com/document/d/1MCGTzVEdNqUTrYqNq6ZMbBsYmLdD1oqQa7mCu6qSH9c/edit#heading=h.98f72y6frg8v).